agb/src/agb_alloc/mod.rs

```diff
@@ -42,8 +43,17 @@ static GLOBAL_ALLOC: BlockAllocator = unsafe {
     })
 };
 
+pub static EWRAM_ALLOC: &BlockAllocator = &GLOBAL_ALLOC;
+pub static IWRAM_ALLOC: &BlockAllocator = &__IWRAM_ALLOC;
```

Comment:
Could you put some doc comments on these with examples of how to use them?

Reply:
Have done so, do you think they are clear?

Reply:
They'll do for now until I do my big documentation pass :)